submodules/eps/__init__.py

```diff
@@ -1,68 +1,140 @@
 import logging
 import time
 import smbus
+import yaml
 
 from smbus2 import SMBusWrapper
 from functools import partial
 
 from core.mode import Mode
 from core.threadhandler import ThreadHandler
+from core.error import Error
+from core.log import Log
+from submodules import telemetry
 
 # Initialize global variables
 logger = logging.getLogger("EPS")
 address = 0x57
 epsdict = {'a': 1, 'i2c': 2, 'c': 3, 'antenna': 4,
            'pi': 5, 'iridium': 6, 'aprs': 7, 'h': 8, 'i': 9, 'j': 10}
-mode = Mode.NORMAL
-
+with open('config/config_default.yml') as stream:
```

Comment:
During the startup sequence, `core` reads from the `config_defaul.yml` file and loads it into memory by storing it inside a global variable in `core` called `config`. Reloading the `config_default.yml` file in each module would be a waste of memory and also removes the functionality of having a `config_custom.yml` file during development. Instead, use `from core import config` and reference the `config` variable when you need to read configuration data